package.json

```diff
@@ -33,6 +33,7 @@
         "jasmine-core": "^3.6.0",
         "jasmine-jquery": "2.1.1",
         "karma": "^6.1.1",
+        "karma-brief-reporter": "^0.2.1",
```

Comment:
This shows a brief summary of the tests and info on the failed tests at the end. I find it much more useful than the mocha reporter and use it locally.